src/PumpStream.php

```diff
@@ -67,7 +67,7 @@ public function close()
 
     public function detach()
     {
-        $this->tellPos = false;
+        $this->tellPos = 0;
```

Comment:
Note that I changed this.. it should always be int

Reply:
👍